app/assets/stylesheets/custom.css.scss

```diff
@@ -152,3 +152,36 @@ p {
     color: #333;
   }
 }
+
+/* forms */
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
+  width: 100%;
+  margin-bottom: 15px;
+  @include box_sizing;
```

Comment:
Name of mixin `box_sizing` should be written in all lowercase letters with hyphens instead of underscores